analysis/taint/testdata/basic/config.yaml

```diff
@@ -1,20 +1,20 @@
 taint-tracking-problems:
  - # The package regex matches all possible ways the package name might appear depending on how the program is loaded
   sources:
-    - package: "(main)|(command-line-arguments)"
+    - package: "(basic)|(main)|(command-line-arguments)"
```

Comment:
Is that the name of the package that will be used when program is loaded in tests or just added for safety?